api/leveldb/leveldb_wt.cc

```diff
@@ -248,12 +248,12 @@ leveldb::DB::Open(const Options &options, const std::string &name, leveldb::DB *
     s_conn << "exclusive,";
 #ifndef HAVE_BUILTIN_EXTENSION_SNAPPY
   if (options.compression == kSnappyCompression)
-    s_conn << "extensions=[libwiredtiger_snappy.so],";
+    s_conn << "extensions=[" EXTENSION_PATH "snappy/.libs/libwiredtiger_snappy.so],";
```

Comment:
I believe this change will break the code when running against an installed version of WiredTiger.